scaffolding-chef-infra/lib/scaffolding.ps1

```diff
@@ -96,8 +96,19 @@ if(!`$env:CFG_SSL_VERIFY_MODE){
     `$env:CFG_SSL_VERIFY_MODE = "verify_peer"
 }
 
+`$env:CFG_CHEF_LICENSE = "{{cfg.chef_license.acceptance}}"
+if(!`$env:CFG_CHEF_LICENSE){
+    `$env:CFG_CHEF_LICENSE = "undefined"
+}
+
+if(`$env:CFG_CHEF_LICENSE -eq "undefined"){
```

Comment:
I don't think this evaluation should be done based on the value of the license_acceptance configuration but off the chef-client version though I'm not sure how to do that.

Reply:
I like the concept so that we have more granular control on the chef-client command